.github/workflows/build.yml

```diff
@@ -1,5 +1,9 @@
 name: Package builds
 on: [push, pull_request]
+# Only build for latest push/PR unless it's main or release/
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' || startsWith( github.ref, 'refs/heads/release/' ) }}
```

Comment:
@rocodes pointed out that this logic is wrong, I think it should be `&& !startsWith(...)`

Reply:
Fixed and documented the expected logic in the commit message / PR description